docs/how-to/custom-artifact.rst

```diff
@@ -70,17 +70,19 @@ additional metadata to capture, this is where we would capture it
         def construct(
             cls,
             name: str,
-            value: Optional[Any] = None,
-            fname: Optional[str] = None,
-            created_at: Optional[datetime] = None,
-            writer_kwargs: Optional[dict] = None,
+            value: Any | None = None,
```

Comment:
FYI: `typings.Any` [includes](https://docs.python.org/3.9/library/typing.html#the-any-type) `None`.

Reply:
Done.